stock_move_location/wizard/stock_move_location_line.py

```diff
@@ -22,7 +39,9 @@ class StockMoveLocationWizardLine(models.TransientModel):
         string="Origin Location", comodel_name="stock.location"
     )
     destination_location_id = fields.Many2one(
-        string="Destination Location", comodel_name="stock.location"
+        string="Destination Location",
+        comodel_name="stock.location",
+        compute="_compute_destination_location",
```

Comment:
I would say it should be stored, readonly False in order to allow user change one line if desired

Reply:
Missing readonly=False

Reply:
I couldn't implement the readonly=False funtionality because compute override the destination location in every line. I try to do that later.